docs/farming-&-staking/farming/advanced-cli/platforms/_ubuntu.mdx

```diff
@@ -1,94 +1,135 @@
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-import CodeBlock from '@theme/CodeBlock';
-import Link from '@docusaurus/Link';
 import styles from '@site/src/pages/index.module.css';
 
-## Step 1: Download the Advanced CLI Executables
+import Card from '@site/src/components/Card';
+import CardBody from '@site/src/components/Card/CardBody';
+import CardFooter from '@site/src/components/Card/CardFooter';
+import CardHeader from '@site/src/components/Card/CardHeader';
+import CardImage from '@site/src/components/Card/CardImage';
+import Column from '@site/src/components/Column';
+import Columns from '@site/src/components/Columns';
+
+<br />
+### Step 1: Download Executables {#download-binaries}
+---
+
+<br />
+<Columns>
```

Comment:
In addition to improving styling, I believe incorporating card elements and columns offers much greater flexibility for organizing guides, enhancing both navigation and the reader's experience. For example, the side-by-side grouping of binaries in `farming/advanced-cli/install` demonstrates how this layout can improve clarity. I had initially considered grouping the existing Link component, but couldn't achieve the same clean and effective design that cards provide.

![image](https://github.com/user-attachments/assets/d2451568-a86f-4f3a-bce5-94b3970245dd)

It seems there may be a way to hide tags in the Crowdin editor that would work. Could this [approach](https://community.crowdin.com/t/how-can-i-hide-tags-in-the-editor/278/2) or [this one](https://community.crowdin.com/t/yaml-style-path-exclusion/4563/2) be helpful? I’m also curious if this method could be applied to prevent slugs from being translated.

Reply:
> It seems there may be a way to hide tags in the Crowdin editor that would work. Could this [approach](https://community.crowdin.com/t/how-can-i-hide-tags-in-the-editor/278/2) or [this one](https://community.crowdin.com/t/yaml-style-path-exclusion/4563/2) be helpful? I’m also curious if this method could be applied to prevent slugs from being translated.

I believe I've read all possible forum and Reddit posts about all the crowdin parsing issues and been in back-and-forth with their support, but none of these settings worked for hiding 100% of HTML tags and a list of other elements. It worked for some, but we still encounter inconsistencies here and there.